crates/sui-types/src/gas.rs

```diff
@@ -473,7 +476,7 @@ impl<'a> SuiGasStatus<'a> {
         let storage_cost =
             NumBytes::new(new_size as u64).mul(*self.cost_table.storage_per_byte_cost);
         self.deduct_storage_cost(&storage_cost).map(|gu| {
-            self.storage_rebate += storage_rebate;
+            self.storage_rebate.add_assign(storage_rebate);
```

Comment:
How would someone in the future know to make this change instead of relying on the macro?